lib/stripe/credit_card_clone_destroyer.rb

```diff
@@ -17,7 +17,9 @@ def destroy_clones(card)
         next unless customer_id
 
         customer = Stripe::Customer.retrieve(customer_id, stripe_account: stripe_account)
```

Comment:
we are reusing variables here :see_no_evil: 
customer is a ofn customer above and here we becomes a stripe customer. this variable should be named stripe_customer, not customer. can you please improve this? Thanks!

Reply:
🙈 😲 👍